Tests/Data/Parabolic/ComponentTransport/goswami/goswami_input.prj

```diff
@@ -1,284 +1,226 @@
-        <parameter>
-            <name>kappa2</name>
-            <type>Constant</type>
-            <values>1e-3 0 0 1e-3</values>
-        </parameter>
-    </parameters>
-    <process_variables>
-        <process_variable>
-            <name>concentration</name>
-            <components>1</components>
-            <order>1</order>
-            <initial_condition>c0</initial_condition>
-            <boundary_conditions>
-                <boundary_condition>
-                    <geometrical_set>goswami_mesh</geometrical_set>
-                    <geometry>left</geometry>
-                    <type>Dirichlet</type>
-                    <parameter>c_Dirichlet_left</parameter>
-                </boundary_condition>
-                <boundary_condition>
-                    <geometrical_set>goswami_mesh</geometrical_set>
-                    <geometry>right</geometry>
-                    <type>Dirichlet</type>
-                    <parameter>c_Dirichlet_right</parameter>
-                </boundary_condition>
-            </boundary_conditions>
-        </process_variable>
-        <process_variable>
-            <name>pressure</name>
-            <components>1</components>
-            <order>1</order>
-            <initial_condition>p0</initial_condition>
-            <boundary_conditions>
-                <boundary_condition>
-                    <geometrical_set>goswami_mesh</geometrical_set>
-                    <geometry>topleft</geometry>
-                    <type>Dirichlet</type>
-                    <parameter>p_Dirichlet_topleft</parameter>
-                </boundary_condition>
-                <boundary_condition>
-                    <geometrical_set>goswami_mesh</geometrical_set>
-                    <geometry>topright</geometry>
-                    <type>Dirichlet</type>
-                    <parameter>p_Dirichlet_topright</parameter>
-                </boundary_condition>
-            </boundary_conditions>
-        </process_variable>
-    </process_variables>
-    <nonlinear_solvers>
-        <nonlinear_solver>
-            <name>basic_picard</name>
-            <type>Picard</type>
-            <max_iter>300</max_iter>
-            <linear_solver>general_linear_solver</linear_solver>
-        </nonlinear_solver>
-    </nonlinear_solvers>
-    <linear_solvers>
-        <linear_solver>
-            <name>general_linear_solver</name>
-            <lis>-i bicgstab -p jacobi -tol 1e-11 -maxiter 20000</lis>
-            <eigen>
-                <solver_type>BiCGSTAB</solver_type>
-                <precon_type>DIAGONAL</precon_type>
-                <max_iteration_step>20000</max_iteration_step>
-                <error_tolerance>1e-11</error_tolerance>
-            </eigen>
-            <petsc>
-                <prefix>hc</prefix>
-                <parameters>-hc_ksp_type bcgs -hc_pc_type bjacobi -hc_ksp_rtol 1e-11 -hc_ksp_max_it 20000</parameters>
-            </petsc>
-        </linear_solver>
-    </linear_solvers>
+	<meshes>
```

Comment:
Would be nice to see the original formatting. Four spaces instead of tabs. Then the diff would make more sense, because now it looks like the whole file was rewritten.

Reply:
The tool `xmlstarlet` can format xml files.